cmd/servicelog/post.go

```diff
@@ -337,6 +359,81 @@ func (o *PostCmdOptions) parseUserParameters() {
 	}
 }
 
+// parseOverides parses all the '-o FOO=BAR' overrides which replace items in the final JSON document
+func (o *PostCmdOptions) parseOverrides() (map[string]string, error) {
+	usageMessageError := errors.New("wrong syntax of '-r' flag. please use it like this: '-r FOO=BAR'")
+	overrideMap := make(map[string]string)
+
+	for _, v := range o.Overrides {
+		if !strings.Contains(v, "=") {
+			return nil, usageMessageError
+		}
+
+		param := strings.SplitN(v, "=", 2)
+		if param[0] == "" || param[1] == "" {
+			return nil, usageMessageError
+		}
+
+		overrideMap[param[0]] = param[1]
+	}
+
+	return overrideMap, nil
+}
+
+// applyOverrides applies the overrides to the Message by JSON tag
+func (o *PostCmdOptions) applyOverrides(overrideMap map[string]string) error {
```

Comment:
Is there value in a method with a single loop in it?  Would the for loop be better either one level up in the parent function, calling overrideField directly?